parameter_files/fates_params_default.cdl

```diff
@@ -97,7 +97,7 @@ variables:
 	double fates_allom_hmode(fates_pft) ;
 		fates_allom_hmode:units = "index" ;
 		fates_allom_hmode:long_name = "height allometry function index." ;
-		fates_allom_hmode:possible_values = "1: O'Brien 1995; 2: Poorter 2006; 3: 2 parameter power law; 4: Chave 2014; 5: Martinez-Cano 2019." ;
+		fates_allom_hmode:possible_values = "1: O\'Brien 1995; 2: Poorter 2006; 3: 2 parameter power law; 4: Chave 2014; 5: Martinez-Cano 2019." ;
```

Comment:
```suggestion
		fates_allom_hmode:possible_values = "1: O'Brien 1995; 2: Poorter 2006; 3: 2 parameter power law; 4: Chave 2014; 5: Martinez-Cano 2019." ;
```

Reply:
@jkshuman I'm just going to remove the apostrophe in Obrien's name.  I feel like there is a back and forth on the preference here, and I'm not sure how netcdf interprets that character

Reply:
@rgknox no worries. I had thought that \ character was a mistake. happy to go with your choice on this.

Reply:
ok, sounds good